src/components/organisms/AdminVenueView/components/RunTabView/RunTabView.tsx

```diff
@@ -45,7 +45,7 @@ export const RunTabView: React.FC<RunTabViewProps> = ({ venue }) => {
       </div>
       <div className="RunTabView__main">
         <div className="RunTabView__toolbar RunTabView--spacing">
-          <RunTabToolbar venueId={venueId} />
+          <RunTabToolbar venueId={venueId} venueName={venue.name} />
```

Comment:
Might as well just pass in `venue` instead separate `id` and `name`.
Otherwise `RunTabToolbar` can just use the `venueId` prop to get a hold of the venue itself.

Reply:
I don't want to pass the whole venue object because we need only 2 fields from that object and it's unnecessary.

I can use the `venueId` and get a hold of the venue if that's the approach you prefer.

Reply:
Not really about preference.

Do you think the toolbar might need more info than id and name?  Maybe less?

Not even sure it needs the name only to pass it down to a component inside. This kind of tight props coupling is harder to maintain.

Reply:
Probably not, I don't see it using anything else from the venue object.